tools/scripts/ansible/templates/server-env.j2

```diff
@@ -5,3 +5,7 @@ MYSQL_PASSWORD={{ mysql_users[0].password }}
 MYSQL_DB=cldrdb
 WLP_SKIP_MAXPERMSIZE=true
 LOG_DIR=/var/log/openliberty/cldr
+
+# DataDog params. Will be updated by the deploy script.
+DD_GIT_COMMIT_SHA=0000000
+DD_GIT_REPOSITORY_URL=github.com/unicode-org/cldr
```

Comment:
I don't quite understand the duplication here and in deploy-sh.j2

Reply:
ideally "MONITOR" instead of "DD" for encapsulation, we might eventually have different/multiple monitoring tools using the same config

Reply:
This parameter is also consumed by the datadog agent in this form using these names. It's a "URL" but doesn't include the scheme (https://). 
The duplication is because this file needs an initial value, which the deploy script updates as the comment notes.